### ISSUE TITLE
Is this usable with windows?

### ISSUE THREAD

**Issue by ithr0n** [skvvv/pikapy#10]
It seems to install within a unix environment, is it usable with windows?

**Comment by Lionir** [comment]
@ithr0n Yes, I can confirm that this works on windows. You need to install Python (favorably 2.7.11 rather than 3 due to 2.7.x's popularity) and pip. Maybe Git too, not sure. But it definitely works inside the CMD prompt.

**Comment by djv2** [comment]
can't get this to work in windows. which python script do we run? there are many. i tried them all, none work

**Comment by Lionir** [comment]
@djv2 Follow the read me.

**Comment by djv2** [comment]
@Lionir i do. i am technical too. there is no script nor package that's just named "pikapy"
please copy/paste exact cmd for windows to use, thanks

**Comment by bukubuke** [comment]
You need to use pip and git to install the package then u go and CD the location where pikapy is at.

**Comment by djv2** [comment]
@bukubuke yes i know, like i said i'm technical. already did that. then what? in the pikapy folder is just a bunch of python scripts

**Comment by skvvv** [comment]
join us on, ill try to help

https://discordapp.com/channels/212586752127270912/212586752127270912

**Comment by ithr0n** [comment]
Sorry for reopening this thing, but I found it out myself yesterday... the application is installing into C:\Python27\Scripts

This directory should be appended to your systems environment path configuration. Side advantage: pip is also usable directly without "python -m ...".

Or you start the application directly from that directory.

Nevertheless, I got errors yesterday and decided to stop further investigation, because I found a alternative for me.

Side node: If you would implement usage of random ip proxys while creating the accounts and automatic email validation bot, this would be great :)
